packages/vuetify/src/styles/generic/_transitions.scss

```diff
@@ -49,6 +49,7 @@
   &-enter-active,
   &-leave-active {
     transition-property: transform, opacity;
+    pointer-events: none;
```

Comment:
Is this necessary with the adding/removing of pointer-events with styles?

Reply:
The hooks only run if there's a target. Most transitions should probably have this tbh